plonky2x/core/src/frontend/hash/sha/curta/builder.rs

```diff
@@ -42,22 +43,22 @@ impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
         }
 
         // Prove correctness of the digest using the proof hint.
```

Comment:
nit: This should be the function description above `curta_constrain_sha` in a doc comment.